engine/src/main/resources/org/terasology/include/globalFunctionsFragIncl.glsl

```diff
@@ -147,16 +147,15 @@ float calcPcfShadowTerm(sampler2D shadowMap, float lightDepth, vec2 texCoord, fl
 float calcVolumetricFog(vec3 fogWorldPosition, float volumetricHeightDensityAtViewer, float globalDensity, float heightFalloff) {
     vec3 cameraToFogWorldPosition = -fogWorldPosition;
 
-    float fogInt = length(cameraToFogWorldPosition) * volumetricHeightDensityAtViewer;
+    float fogTotal = length(cameraToFogWorldPosition) * volumetricHeightDensityAtViewer;
```

Comment:
`fogTotal` sounds a bit strange, especially since a big part of the code-philosophy here is "english-like code". `totalFog` sounds kinda better, but I'm wondering if we can do better.

Reply:
What I mean by it is something like "total amount of fog in that line of sight", although that would also imply globalDensity is included (although actually, maybe it should be). Would "fogAmount" be any better?

Reply:
Think about it this way:
"the fog total is equal to.." vs "the total fog is equal to.."

Yes, fogTotal does indeed sound better.
Ykw? let's wait for manu, he often has better variable name suggestions than me :P

Reply:
A few of ideas:
- `totalFogInLineOfSight ` (more colloquial but less precise, my least favorite)
- `totalFogUntilSample` (compact, more precise)
- `totalFogBetweenCameraAndSample` (very precise, verbose, perhaps too much)
- `totalFogFromCameraToSample` (a good compromise?)

We could also drop the total and use something like `fogValueFromCameraToSample`.

Reply:
`fogBetweenCameraAndSample` ?
I kinda agree it's verbose, but it's not the 90s anymore ;)

Reply:
Even if it's not the 90's, I (at least) am likely to gloss over excessively long variable names and find them more effort to read (this probably contributed to me confusing invProjMatrix and invViewProjMatrix). Unless there are further objections, I'm going with "totalFogToSample" ("until" sound too much like something time-related to me). I don't think there are any good options for many of these variable names, but the code is sufficiently simple that it should make their purpose clear anyway.

Reply:
Over the long variable names: it's certainly a style thing. I do encourage them as long as they make sense to limit ambiguities - obviously they don't always make sense and when possible shorter names are perfectly valid.